conversiontracerbullet/src/main/java/org/stanford/AuthDBProperties.java

```diff
@@ -19,22 +16,22 @@
 
     private static Logger log = LogManager.getLogger(AuthDBProperties.class.getName());
 
-    private static final String DEFAULT_CONFIG_FILE = "/server.conf";
+    private static final String PROPERTY_RESOURCE = "/server.conf";
 
     private String server = null;
     private String service = null;
     private String userName = null;
     private String userPass = null;
-    private String propertyFile = null;
+    private Properties properties = null;
```

Comment:
Oh, yes, that's possible.  The next consideration is how easy the class is to test.  As it is, those methods are easier to test (or use in tests).  I guess that's why it is this way, rather than those loading methods be void methods.  At some point while developing tests, it was easier if they return Properties.  It's reasonable to request this, but it could entail more work to change tests as well.  Not sure we really gain much?  Perhaps we can pair on this rather than play github comment tennis?